omero/developers/Web/CreateApp.rst

```diff
@@ -14,27 +14,50 @@ All official OMERO applications can be installed from PyPI_.
 Getting set up
 --------------
 
-In order to deploy OMERO.web in a development or testing environment please 
+In order to deploy OMERO.web in a development or testing environment,
+you can use Docker as described below (recommended) or 
 follow the instructions under :doc:`Deployment`.
 
-You should make sure that you can access the webclient on
-your local machine before starting to develop your own code. Be sure to
-use the correct port number, e.g.
+If you want to make changes to the OMERO.web code itself, go to
+:doc:`/developers/Web/EditingOmeroWeb`.
 
--  `http://localhost:4080/webclient/ <http://localhost:4080/webclient/>`_
+Use the app template
+--------------------
 
-When you edit and save your app, Django will automatically detect this and you
-only need to refresh your browser to see the changes.
+Go to `https://github.com/will-moore/omero-web-apps-examples <https://github.com/will-moore/omero-web-apps-examples>`_.
```

Comment:
preferable to push minimal web-app to the parent repo that pointing to your repo in the official doc

Reply:
That's what I meant in the description by "NB: need to rename https://github.com/will-moore/omero-web-apps-examples/ to https://github.com/ome/omero-web-apps-examples/ when ome/omero-web-apps-examples#3 is merged."

Reply:
Sorry I opened the ``Files changed`` tab without checking description. My bad!